src/vm/wren_vm.c

```diff
@@ -1621,3 +1622,46 @@ void wrenGetVariable(WrenVM* vm, const char* module, const char* name,
   
   setSlot(vm, slot, moduleObj->variables.data[variableSlot]);
 }
+
+// Prints to WrenVM.errorFn if provided, otherwise prints to stderr.
+void wrenPrintError(WrenVM* vm, const char* format, ...)
+{
+  va_list args;
+  
+#if defined( _MSC_VER )
+  // MSVC doesn't have snprintf():
+  #define WREN_STRING_SIZE(format, args) _vscprintf(format, args)
+  #define WREN_STRING_PRINT(buffer, size, format, args) _vsnprintf_s(buffer, size, _TRUNCATE, format, args)
+#else
+  // Use C99's snprintf():
+  #define WREN_STRING_SIZE(format, args) vsnprintf(NULL, 0, format, args)
+  #define WREN_STRING_PRINT(buffer, size, format, args) vsnprintf(buffer, size, format, args)
+#endif
+
+  // Allocate `buffer` to hold the output:
+  va_start(args, format);
+  int sizeNeeded = WREN_STRING_SIZE(format, args);
+  int sizeAllocated = sizeNeeded + 1; // +1 for the null terminator
+  char *buffer = (char *)malloc(sizeAllocated);
```

Comment:
Lets use the reallocate function provided in the config for this, so it goes through the user-provided allocator.